querying/github.go

```diff
@@ -167,6 +173,24 @@ func (gh *GithubDataSource) processRepoFindings(projects *ProjectCollection, rep
 			finding.Severity = githubSeverities[vuln.SecurityVulnerability.Severity]
 		}()
 	}
+
+	if repo.VulnerabilityAlerts.PageInfo.HasNextPage {
+		var repositoryQuery repositoryQuery
+		queryVars := map[string]interface{}{
+			"repoName":    githubv4.String(repo.Name),
+			"orgName":     githubv4.String(gh.orgName),
+			"alertCursor": githubv4.String(endCursor),
+		}
+		err := gh.GhClient.Query(gh.ctx, &repositoryQuery, queryVars)
+		if err != nil {
+			return err
+		}
+
+		nextCursor := repositoryQuery.Repository.VulnerabilityAlerts.PageInfo.EndCursor
+		log.Info().Any("alertCursor", queryVars["alertCursor"]).Msg("Querying for more vulnerabilities for a repository.")
+		return gh.processRepoFindings(projects, repositoryQuery.Repository, nextCursor)
```

Comment:
I have verified this locally, will add some unit test for this in a follow up PR.